addons/project/models/project_task.py

```diff
@@ -937,13 +938,39 @@ def create(self, vals_list):
             # if the portal user could really create the tasks based on the ir rule.
             tasks.with_user(self.env.user).check_access_rule('create')
         current_partner = self.env.user.partner_id
+
+        all_partner_emails = []
+        for task in tasks:
+            all_partner_emails += tools.email_split(task.email_cc)
+        partners = self.env['res.partner'].search([('email', 'in', all_partner_emails)])
+        partner_per_email = defaultdict()
+        for partner in partners:
+            if not all(u.share for u in partner.user_ids):
+                partner_per_email[partner.email] = partner
         for task in tasks:
             if task.project_id.privacy_visibility == 'portal':
                 task._portal_ensure_token()
             for follower in task.parent_id.message_follower_ids:
                 task.message_subscribe(follower.partner_id.ids, follower.subtype_ids.ids)
             if current_partner not in task.message_partner_ids:
                 task.message_subscribe(current_partner.ids)
+            if task.email_cc:
+                partners_with_internal_user_ids = []
+                for email in tools.email_split(task.email_cc):
+                    if partner_per_email.get(email, False):
+                        partners_with_internal_user_ids += [partner_per_email[email].id]
+                if not partners_with_internal_user_ids:
+                    continue
+                task.message_subscribe(partners_with_internal_user_ids)
+                message_values = {
+                    'subject':  Markup(_('Invitation to follow Task: %(task_name)s') % {'task_name': task.name}),
+                    'body': Markup(_('<p>Hello,</p><p>You have been invited to follow Task document: %(task_name)s</p>') % {'task_name': task.name}),
+                    'record_name':  Markup(task.name),
```

Comment:
Hello @odoo/rd-security ,

Can we have a override?

We use Markup function for the content of a message their could contain HTML. 🙂
I am not sure if it is really useful for the content of `record_name` and I am not if it is correctly written. 🤷🏼‍♂️

Thanks in advance for your review. 🙂 

Have a nice evening! 🙂